static/js/compose_closed_ui.js

```diff
@@ -64,17 +64,63 @@ function update_conversation_button(btn_text, title) {
     $("#left_bar_compose_private_button_big").prop("title", title);
 }
 
-function update_buttons(text_stream) {
+function update_buttons(text_stream, disable_reply, title_reply) {
     const title_stream = text_stream + " (c)";
     const text_conversation = $t({defaultMessage: "New private message"});
     const title_conversation = text_conversation + " (x)";
+    const text_deactivated_conversation = $t({defaultMessage: "Reply selected message"});
+    let title_deactivated_conversation = text_deactivated_conversation + " (r)";
+    if (narrow_state.pm_ids_string()) {
+        const user_ids = people.user_ids_string_to_ids_array(narrow_state.pm_ids_string());
+        const user_ids_dicts = user_ids.map((user_id) => ({id: user_id}));
+        if (people.verify_non_active_human_id(user_ids_dicts[0].id)) {
```

Comment:
This could be a group PM too, right? In that case, we should check if any of the users are deactivated and give an appropriate message stating exactly who is deactivated. Also, we don't need to call `narrow_state.pm_ids_string()` twice here.